<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogImageSlice</class>
 <widget class="QDialog" name="DialogImageSlice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Image Slice</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QComboBox" name="cmbSliceType">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>horizontal slice</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>vertical slice</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>custom slice</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabelSlice" name="lblSlice" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis">
          <string>&lt;h4&gt;Slice of geometry&lt;/h4&gt;</string>
         </property>
         <property name="text" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblGeometrySlice">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Slice of geometry:</string>
         </property>
         <property name="buddy">
          <cstring>cmbSliceType</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QCheckBox" name="chkAspectRatio">
         <property name="text">
          <string>Maintain &amp;aspect ratio</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="chkSmoothInterpolation">
         <property name="text">
          <string>Smooth &amp;interpolation</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="lblSlicedImage">
         <property name="text">
          <string>Species concentrations along slice vs time:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabelMouseTracker" name="lblImage" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis">
          <string>&lt;h4&gt;Species concentrations along slice vs time&lt;/h4&gt;</string>
         </property>
         <property name="text" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="lblMouseLocation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>mouse location: (x=., y=., t=.)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLabelMouseTracker</class>
   <extends>QWidget</extends>
   <header>qlabelmousetracker.hpp</header>
  </customwidget>
  <customwidget>
   <class>QLabelSlice</class>
   <extends>QWidget</extends>
   <header>qlabelslice.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
